src/shared/client/cg_api.cpp

```diff
@@ -554,9 +558,12 @@ void trap_R_GetTextureSize( qhandle_t handle, int *x, int *y )
 
 qhandle_t trap_R_GenerateTexture( const byte *data, int x, int y )
 {
+	size_t size = x * y * 4;
+	if (!size) { return 0; }
+
 	qhandle_t handle;
-	std::vector<byte> mydata(x * y * 4);
-	memcpy(mydata.data(), data, x * y * 4 * sizeof( byte ) );
+	std::vector<byte> mydata(size);
```

Comment:
The size is now wrong.

Reply:
This code is not modified anymore, I just added an ASSERT before running it instead.